web/templates/message_body.hbs

```diff
@@ -1,30 +1,34 @@
 {{#if include_sender}}
     {{> message_avatar . ~}}
 {{/if}}
-<span class="message_sender">
+<div class="message_sender">
     {{#if include_sender}}
-        <span class="sender_info_hover sender_name" role="button" tabindex="0">
-            <span class="view_user_card_tooltip sender_name_text" data-is-bot="{{sender_is_bot}}">
+    <div class="sender_info">
```

Comment:
You're introducing this new class with new flex behavior and a gap, and it's going to impact all kinds of senders. If this is the right solution, then I expect to see a bunch of screenshots showing status emoji, etc.

I also note you're adding noise here, still, with more <div> to <span> conversions. That was supposed to have been taken care of in your first commit.

These kinds of matters are why I avoid reviewing your work, because I do not trust that you're taking feedback to heart, or ensuring that your changes are not impacting other parts of the UI.